LICENSE

```diff
@@ -865,6 +914,135 @@ License (https://github.com/mooreds/gwt-crypto/blob/master/bouncycastle-lic.txt)
 Copyright (c) 2000 - 2006 The Legion Of The Bouncy Castle (http://www.bouncycastle.org)
 Licensed under the MIT License (MIT)
 
+--------------------------------------------------------------------------------------------------------
+for packages/stunner-editors/errai-common/src/main/java/org/jboss/errai/common/client/util/TimeUnit.java
+
+Written by Doug Lea with assistance from members of JCP JSR-166
+Expert Group and released to the public domain, as explained at
+http://creativecommons.org/licenses/publicdomain
+
+Creative Commons Legal Code
```

Comment:
It is addressed by https://github.com/apache/incubator-kie-tools/pull/2777/files